dev.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: kubernetes-deploy
+up:
+  - ruby: 2.3.3
+  - bundler
+  - homebrew:
+    - Caskroom/cask/minikube
+  - custom:
+      name: Minikube Cluster
+      met?: test $(minikube status | grep Running | wc -l) -eq 2 && $(minikube status | grep -q 'Correctly Configured')
+      meet: minikube start --vm-driver=xhyve --kubernetes-version=v1.7.5
+      down: minikube stop
```

Comment:
FWIW the reason I used `minikube delete` elsewhere was that with `stop` I had noticed on start `minikube` would not switch namespaces - but I think it was a red herring, it appears to work properly here.